### ISSUE TITLE
osbuilder: change cbl-mariner image from initrd-builder to image-builder format

### ISSUE THREAD

**Issue by danmihai1** [kata-containers/kata-containers#8815]
Reasons:

1. Consistency with the cbl-mariner image format from AKS Confidential Containers.
2. The new image-builder format image will be useful for Kata CI to test the new dm-verity format from https://github.com/kata-containers/kata-containers/pull/7988.

**Comment by sprt** [comment]
@danmihai1 We had fixed this, correct?

**Comment by danmihai1** [comment]
Yes, cbl-mariner CI tests are using a virtio-blk rootfs image nowadays.
